.github/workflows/deploy.yml

```diff
@@ -71,12 +80,47 @@ jobs:
         # pytest -m "not accessibility" runs every test which is not marked 
         # accessibility.
         run: python -m pip install pytest && python -m pytest -m "not accessibility" .
+        
+  testing-unit-postgres: 
+    runs-on: ubuntu-latest
+    environment: Dev
+    if: ${{ inputs.postgres_unit_testing == true }}
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: fsd_app_store_test
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # Maps tcp port 5432 on service container to the host
+          - 5432:5432
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.10.1
+      - name: install dependencies
+        run: python -m pip install --upgrade pip && python -m pip install -r requirements-dev.txt
+      - name: run unit tests
+        # pytest -m "not accessibility" runs every test which is not marked 
+        # accessibility.
```

Comment:
Should we still be excluding accessibility tests here? If not pls remove the comment :)